templates/docker-container.yml

```diff
@@ -6,14 +6,35 @@
 trigger:
 - master
 
-pool:
-  vmImage: 'ubuntu-latest'
+resources:
+- repo: self
 
 variables:
-  imageName: 'your-container-image-name:$(build.buildId)'
-  dockerfile: '{{ dockerfilePath }}'
-  buildContext: '{{ buildContextPath }}'
+  # Container registry service connection established during pipeline creation - Docker Hub/ACR/...
```

Comment:
If this template and the accompanying configuration panel just address building and pushing an image to existing ACR, remove Docker Hub and ... from this line